src/oop_ext/foundation/callback/_tests/test_callback.py

```diff
@@ -643,57 +643,6 @@ def After(*args):
         c()
         assert self.called == 1
 
-    def testHandleErrorOnCallback(self, mocker):
```

Comment:
There is other test that check that errors on callback will be triggered? I think you just need to remove the part of HandleException, the other should be kept